🎉(global) bootstrap project (#1)
openfun/vernajs: README.md

# Verna.js 🏗
[![CircleCI](https://circleci.com/gh/openfun/vernajs/tree/master.svg?style=svg)](https://circleci.com/gh/openfun/vernajs/tree/master)

An extensible form builder based on [React JSON Schema Form](https://github.com/rjsf-team/react-jsonschema-form).

## Overview

> Verna.js is a React component using [JSON Schema](http://json-schema.org/) to build and
> generate forms. As it relies on [React JSON Schema Form](https://github.com/rjsf-team/react-jsonschema-form)
> library, it is fully extensible to fit your needs.

Reviewer: I would say it uses json schema but generates json schema as output...
```suggestion
Verna.js is a React component to build forms that generates a [JSON Schema](http://json-schema.org/) description of the form as output. The form can then be rendered with the [React JSON Schema Form](https://github.com/rjsf-team/react-jsonschema-form)
library, which makes it fully extensible to fit your needs.
```

## Quick preview

> You can check examples available within `examples` folder. Take a loot at `README.md` in

Reviewer: ```suggestion
You can check examples available within the `examples` folder. Take a loot at the `README.md` file in
```

each example then follow instructions to run it.

### Instructions

🚧 **`@openfun/vernajs` is not yet available on NPM. Stay tuned !**
Soon instructions to install `@openfun/vernajs` within your project.

To run the project, you have to run :
1. `npm install` or `yarn`
2. `npm start` or `yarn start`

> You should now be able to access to the playground example from http://localhost:3000

Reviewer: ```suggestion
You should now be able to access the playground example from http://localhost:3000
```
